src/main/resources/org/kuali/kfs/gl/spring-gl.xml

```diff
@@ -57,7 +57,8 @@
 				<value>posterJob</value>
 				<value>posterBalancingJob</value>
 				<value>sufficientFundsAccountUpdateJob</value>
-				<value>clearPendingEntriesJob</value>
+                <value>sufficientFundsFullRebuildJob</value>
```

Comment:
so I think what we have here is a tabs vs. spaces thing; it looks like the other lines use tabs, and the new line I added is using spaces

There is an editor option in Intellij to use tabs instead of spaces; mine is currently unchecked and I’m hesitant to check it

something we need to figure out, but maybe not today